### ISSUE TITLE
Not working with NS 7

### ISSUE THREAD

**Issue by marcocasadio** [davecoffin/nativescript-filterable-listpicker#59]
I'm using this plugin but after upgrade to NativeScript 7 it doesn't work anymore.
Are you going to update it?

I'm testing it on iOS running in Simulator.
CLI: 7.0.8

thanks

**Comment by crowmagnumb** [comment]
I concur, I get the the error `Class constructor GridLayout cannot be invoked without 'new'`.

**Comment by crowmagnumb** [comment]
@marcocasadio nstudio has an updated version of this from their fork of this prouect. They are the one's that have taken over management of the NativeScript codebase itself so you are good hands with them. I tried theirs and it works with NS 7. @nstudio/nativescript-filterable-listpicker

**Comment by marcocasadio** [comment]
@crowmagnumb ~3.0.0 is what to use. now it works for me

**Comment by crowmagnumb** [comment]
Yes, you mean of the @nstudio one right? That's the one I found at 3.0.0.

**Comment by marcocasadio** [comment]
yes https://www.npmjs.com/package/@nstudio/nativescript-filterable-listpicker

**Comment by davecoffin** [comment]
Yes, sorry ill update the readme that ive moved this over to the nstudio repo.
